caffeinated/admin/new/pricing/templates/event_tickets_datetime_price_type_base.template.php

```diff
@@ -13,18 +13,18 @@
 ?>
 
 <input type="hidden"
-       name="edit_prices[<?php echo $tkt_row; ?>][<?php echo $PRC_order; ?>][PRT_ID]"
+       name="edit_prices[<?php echo esc_attr($tkt_row); ?>][<?php echo esc_attr($PRC_order); ?>][PRT_ID]"
        class="edit-price-PRT_ID"
-       value="<?php echo $PRT_ID; ?>"
+       value="<?php echo esc_attr($PRT_ID); ?>"
```

Comment:
`$tkt_row`, `$PRC_order`, and `$PRT_ID` are all integers and can therefore use `absint()`